### PR TITLE
Add constructor doc bloc for UploadedFile

### PR DESCRIPTION
**Pull request by settermjd**
I was going through the class and thought it might help others to have the doc bloc added.

### CODE REVIEW

File: src/UploadedFile.php [hunk 1]
```diff
@@ -51,6 +51,14 @@ class UploadedFile implements UploadedFileInterface
      */
     private $stream;
 
+    /**
+     * @param string|resource $streamOrFile
+     * @param int $size
+     * @param int $errorStatus
+     * @param string $clientFilename
```
Comment (samsonasik) [inline src/UploadedFile.php]: @param string|null $clientFilename

File: src/UploadedFile.php [hunk 2]
```diff
@@ -51,6 +51,14 @@ class UploadedFile implements UploadedFileInterface
...
+     * @param string $clientMediaType
```
Comment (samsonasik) [inline src/UploadedFile.php]: @param string|null $clientMediaType

**Comment by settermjd** [comment]
Thanks @samsonasik. I'll get those changes made.

**Comment by weierophinney** [comment]
I incorporated feedback during merge.

**Pull request merged**